src/clj/puppetlabs/services/jruby_pool_manager/jruby_schemas.clj

```diff
@@ -59,14 +59,17 @@
         values are `:jit`, `:force`, and `:off`.  Defaults to `:off`.
 
     * :max-active-instances - The maximum number of JRubyInstances that
-        will be pooled."
+        will be pooled.
+
+    * :environment-vars - The whitelisted environment variables to be passed"
```

Comment:
This docstring is sort of confusing. When reading it, I think environment-vars will be a list of keys from the current environment to pass on to the jruby. However, the code below makes it look like it will be a set of predefined key/values.